templates/page.html

```diff
@@ -286,6 +284,12 @@ <h6>27 December 2019</h6>
         <div class="events-list-container">
           <table class="events-list">
             <tbody>
+              <tr>
+                <td class="events-list-header"><i class="fas fa-video event-conference"></i></td>
+                <td>
+                  Presented "Analysis Reproducibility with REANA on Kubernetes" as part of the CERN Cloud Containers webinars initiative. <a href="https://www.youtube.com/watch?v=EI-8IVOeRUE">[webinar]</a> <a href="https://indico.cern.ch/event/925954/">[event]</a></a>
```

Comment:
Better to say "... webinar series".

Please also amend the link escience2019.sdsc.edu since it times out and CI fails...